public/app/plugins/datasource/loki/querySplitting.ts

```diff
@@ -179,19 +173,45 @@ export function runPartitionedQueries(datasource: LokiDatasource, request: DataQ
   const [instantQueries, normalQueries] = partition(queries, (query) => query.queryType === LokiQueryType.Instant);
   const [logQueries, metricQueries] = partition(normalQueries, (query) => isLogsQuery(query.expr));
 
+  const oneDayMs = 24 * 60 * 60 * 1000;
+  const rangePartitionedLogQueries = groupBy(logQueries, (query) =>
+    query.experimentalChunkRange && isValidDuration(query.experimentalChunkRange)
```

Comment:
If `experimentalChunkRange` is not a valid duration, I would stop and return immediately, instead of defaulting to 1 day.

Reply:
I think it's safe to just remove the check, because we already do it when we set `experimentalChunkRange`: https://github.com/grafana/grafana/blob/svennergr/chunk-range-option/public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilderOptions.tsx#L47-L50

Removed in bd3cfbc163.